lib/public/views/Runs/Details/index.js

```diff
@@ -86,37 +82,42 @@ const runDetails = (model) => {
         };
 
         return [
-            h('h2.mv2', { onremove: () => model.runs.clearRun() }, `Run #${data.payload.runNumber}`),
-            h('.flex-row.mv2', [
-                h('button.btn.btn-primary.w-15.mh2#create', {
-                    onclick: () => model.router.go(`/?page=log-create&id=${data.payload.runNumber}`),
-                }, 'Add Log Entries'),
-                h('label.form-check-label.f5.w-11.mh2', 'Add log entries to the run'),
-            ]),
-            h('.flex-row.mv2', [
-                h('button.btn.btn-success.w-15.mh2', {
-                    onclick: () => model.runs.updateRunTags(),
-                    title: 'This action will overwrite existing tags',
-                    disabled: !model.runs.selectedTagsChanged(),
-                }, 'Update Tags'),
-                h('label.form-check-label.f5.w-15.mh2', { for: 'tags' }, [
-                    'Add one or multiple tags to the run. ',
-                    '(CTRL + click to select multiple tags).',
+            h('.flex-row', [
+                h('.flex.w-100', [
+                    h('h2.mv2', { onremove: () => model.runs.clearRun() }, `Run #${data.payload.runNumber}`),
+                    h('.flex-row.mv2', [
+                        h('button.btn.btn-primary.w-15.mh2#create', {
+                            onclick: () => model.router.go(`/?page=log-create&id=${data.payload.runNumber}`),
+                        }, 'Add Log Entries'),
+                        h('label.form-check-label.f5.w-11.mh2', 'Add log entries to the run'),
+                    ]),
+                    h('.flex-row.mv2', [
+                        h('button.btn.btn-success.w-15.mh2', {
+                            onclick: () => model.runs.updateRunTags(),
+                            title: 'This action will overwrite existing tags',
+                            disabled: !model.runs.selectedTagsChanged(),
+                        }, 'Update Tags'),
+                        h('label.form-check-label.f5.w-15.mh2', { for: 'tags' }, [
+                            'Add one or multiple tags to the run. ',
+                            '(CTRL + click to select multiple tags).',
+                        ]),
+                    ]),
+                    h('select#tags-control.form-control.w-30.mh2', {
```

Comment:
I believe it would be better if we increase the width to at least 50% of the parent size

Reply:
Fixed